include/deal.II/lac/sparse_matrix.templates.h

```diff
@@ -967,9 +967,9 @@ SparseMatrix<number>::mmult (SparseMatrix<numberC>       &C,
       // rows in the matrix A, and for each column in a row we add the whole
       // row of matrix B with that row number. This means that we will insert
       // a lot of entries to each row, which is best handled by the
-      // CompressedSimpleSparsityPattern class.
+      // DynamicSparsityPattern class.
       {
-        CompressedSimpleSparsityPattern csp (m(), B.n());
+        DynamicSparsityPattern csp (m(), B.n());
```

Comment:
I think we should make the same `csp->dsp` substitution here as we did in the tutorial programs.